ajax/libs/venn.js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "venn.js",
+  "filename": "venn.min.js",
+  "description": "A javascript library for laying out area proportional venn and euler diagrams",
+  "homepage": "https://github.com/benfred/venn.js/",
```

Comment:
We don't use git repo from GitHub as the homepage.
Let's use `https://benfred.github.io/venn.js/`.
Thanks.